zenml/utils/requirement_utils.py

```diff
@@ -1,7 +1,9 @@
 import re
-
 import subprocess
 import sys
+from zenml.logger import get_logger
```

Comment:
I think the circular dependency lives here @ramitsurana . If you import zenml here than it wont be able to install and might result in an error like https://github.com/maiot-io/zenml/runs/2437806331?check_suite_focus=true

Reply:
Rather do this import at the line 119